runClient.go

```diff
@@ -1,32 +0,0 @@
-package main
-
-import (
-	"fmt"
-
-	"github.com/blics18/SendGrid/client"
-)
-
-func main() {
-	// runClient.go is where we run all of our functions that we write in Client.go. 
-	// If you'd like to test DropTables and Check, just uncomment the functions.
-	// If we decide to make a command-line interface, we'll write it here as well.
-	
-	fmt.Println("Starting Client")
-
-	// populate the MySQL Database
-	db := client.PopulateDB()
```

Comment:
The flow of this seems odd. It looks like `client.PopulateDB` creates a database object and doesn't close it, but we close it here. I think we should have another function `func NewDB() (*sql.DB, error)` that creates the db conn pool. Then if there is no error, you can pass db variable to `PopulateDB`, and `DropTables` functions.